yiisoft/middleware-dispatcher/config/web.php

```diff
@@ -1,13 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-use Yiisoft\Middleware\Dispatcher\MiddlewareFactory;
-use Yiisoft\Middleware\Dispatcher\MiddlewareFactoryInterface;
-use Yiisoft\Middleware\Dispatcher\MiddlewarePipelineInterface;
-use Yiisoft\Middleware\Dispatcher\MiddlewareStack;
-
-return [
-    MiddlewarePipelineInterface::class => MiddlewareStack::class,
```

Comment:
Are you sure it should be removed?

Reply:
If I removed it because it is not in `config/yiisoft/packages`, I thought it was an error.